storage/remote/queue_manager_test.go

```diff
@@ -522,6 +531,7 @@ type TestWriteClient struct {
 	receivedExemplars map[string][]prompb.Exemplar
 	expectedExemplars map[string][]prompb.Exemplar
 	receivedMetadata  map[string][]prompb.MetricMetadata
+	writes            int
```

Comment:
can we give this a better name? `expectedNumWrites` perhaps?

Reply:
I don't think `expectedNumWrites` is the proper term, because it's not a specified expectation, it's how many are counted.

Reply:
yep, sounds good :+1: